docs/api.yml

```diff
@@ -0,0 +1,215 @@
+openapi: "3.0.0"
+info:
+  title: RT Dashboard API
+  version: 1.0.0
+
+paths:
+  /history.json:
+    get:
+      summary: List history entries
+      responses:
+        200:
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  rows:
```

Comment:
`groups` property is missing

Reply:
Should we specify it if it's going to be removed?

Reply:
added, reflecting https://github.com/djangsters/rt-dashboard/pull/40/files?diff=split&w=1#diff-16147ada3f9dd87580394636cdb94468R116-R117

see below